### ISSUE TITLE
KP.1.1.1 + S:F59S,T22N (18 seqs) with a S:K478T+M153I branch (14) and further S:H445R (2, India, GBW)

### ISSUE THREAD

**Issue by Fearuncle** [sars-cov-2-variants/lineage-proposals#1920]
_This was spotted first by @aviczhl2 in the groups (edited)_

Proposal edited by mod on November 17
KP.1.1.1 > S:F59S (T21738C) > S:T22N (C21627A) >> S:K478T (A22995C) > Orf1a:F2321L  (T7226C) , T11974C ,G19677A>S:M153I (G22021A) > C4898T,  T10084A, S:H445R (A22896G),Orf8:E106* (G28209T)
Query for 22N - 59S branch:  T21738C,C21627A,C20104T,-C6701T,-C25006T,-C1762A,-C25680T,-T22930A (edited on Feb 21)
Samples: 15
Countries: India (GBW), NZL, Singapore
Query for M153I branch: G22021A,T7226C, T11974C Samples: 11 
Query for the new 445R branch : C4898T, G28209T ,T10084A  Samples : 2 (India - GBW) found by @xz-keg 

Tree:
 
<img width="1163" alt="Screenshot 2024-11-17 alle 18 56 13" src="https://github.com/user-attachments/assets/add95ea4-baef-40b5-b446-d8821ac476d2">

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice1_genome_test_2968d_a20ca0.json?label=id:node_7409269

<Details><Summary>
original proposal
</Summary>
![image](https://github.com/user-attachments/assets/7463f5ca-3081-45a5-8507-34eebcb18700)
tree: https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_11c0e_ab6c20.json?c=gt-nuc_19677&gmax=20677&gmin=18677&label=id:node_7072051

Seq: 2024-08-07

Query: G19677A, G22021A

PQ214096.1 PQ214174.1
</details>

**Comment by FedeGueli** [comment]
Thx great catch.

**Comment by FedeGueli** [comment]
Ah but this was spotted by @aviczhl2 a 10ish days ago, then i re found it so it is the third time it has been spotted. i have a query for it wait .. here:G19677A, G22021A  ,T7226C,T3565C

**Comment by aviczhl2** [comment]
No further seqs

**Comment by FedeGueli** [comment]
rearra nging the proposal to just include 59S 22N

**Comment by aviczhl2** [comment]
oh there is another traveler from india.

**Comment by FedeGueli** [comment]
> The tree of this changed including a branch with a reversion at S:F59S>F: <img alt="Screenshot 2024-09-30 alle 12 36 25" width="1161" src="https://private-user-images.githubusercontent.com/87669813/372051679-31e90089-33b0-4bc4-bb82-288d61d31178.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgxNjY3ODAsIm5iZiI6MTcyODE2NjQ4MCwicGF0aCI6Ii84NzY2OTgxMy8zNzIwNTE2NzktMzFlOTAwODktMzNiMC00YmM0LWJiODItMjg4ZDYxZDMxMTc4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA1VDIyMTQ0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5NmIzZWYzNTRmYzc1MDE5YzBkZjRkZDQxY2RiOWQwNWM3YmQxMzgzOTYyMDJhOTIzYTkzNTUyM2NkZTExNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ed74Aqg6cme2xI5_D722jrt_3iJ4nZUncuQty3-WXD8"> https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice2_genome_test_4b64b_a7ba80.json?c=gt-nuc_11704&gmax=12704&gmin=10704&label=id:node_7278672
> 
> Please @Fearuncle keep this checked broadly because i fear some surprise could come from this branch

Now this has been proposed in https://github.com/sars-cov-2-variants/lineage-proposals/issues/2120 
Interestingly the 59S rev branch has a deletion from S:N30-31 S:F32I

**Comment by aviczhl2** [comment]
> Now this has been proposed in #2120 Interestingly the 59S rev branch has a deletion from S:N30-31 S:F32I

Nope it is an usher bug, there's no rev and it is a misplacement,  only 2 of the seqs are actually in this branch, with 22N+59S, and none of them have 478T rev. Suggest to close this.

**Comment by FedeGueli** [comment]
> > Now this has been proposed in #2120 Interestingly the 59S rev branch has a deletion from S:N30-31 S:F32I
> 
> Nope it is an usher bug, there's no rev and it is a misplacement, only 5 of the seqs are actually in this branch, with 22N+59S.
> 
> The latest seq seem uploaded more than 3 weeks ago. GBW from India maybe we can wait for 1 more week or so.

yeah for sure the branch was highlighted to be checked inside this issue because USher placed here and was only three at times. Anyway doesn't count what or who just to keep things together. One of the GBW sequence has 1086 reverted as we saw in another issue i dont recall now which one.

**Comment by aviczhl2** [comment]
> yeah for sure the branch was highlighted to be checked inside this issue because USher placed here and was only three at times. Anyway doesn't count what or who just to keep things together. One of the GBW sequence has 1086 reverted as we saw in another issue i dont recall now which one.

I mean they are separate branches and unrelated.  Usher put together them just because some bugs. The proposed branch here has not appeared for a long time.

**Comment by Fearuncle** [comment]
![image](https://github.com/user-attachments/assets/e5e98425-50cd-40fc-aa0b-48c61cbad43f)
updated

**Comment by xz-keg** [comment]
Only 6 seqs for this branch. Seqs on the S:59 rev branch are unrelated.

**Comment by FedeGueli** [comment]
+1 Singapore

**Comment by xz-keg** [comment]
+2 India

**Comment by FedeGueli** [comment]
> +2 GBW from India, with additional S:M153I, K478T rev and H445R
<img width="1143" alt="Screenshot 2024-11-17 alle 18 06 12" src="https://github.com/user-attachments/assets/5f6e3710-7edc-485e-ab8f-d874d25ccb84">

<img width="1172" alt="Screenshot 2024-11-17 alle 18 03 17" src="https://github.com/user-attachments/assets/01baaa84-ad2b-4685-8fe1-9c6a49001d4b">

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice1_genome_test_2968d_a20ca0.json?label=id:node_7409269

Query for M153I branch: G22021A,T7226C, T11974C
Query for the new 445R branch : C4898T, G28209T  ,T10084A

**Comment by xz-keg** [comment]
There is an XEC branch of C20104T now, T21738C,C21627A,C20104T,-C6701T,-19716 shall be a new query, and it stays at 11.

**Comment by FedeGueli** [comment]
The 478T 153I branch revived in California in January reopening it

**Comment by FedeGueli** [comment]
16 California again, new query:21738C,C21627A,C20104T,-C6701T,-C25006T,-C1762A,-C25680T,-T22930A to exclude a couple of NB.1 from hubei

**Comment by xz-keg** [comment]
no further seqs
